src/boards/include/boards/datanoisetv_rp2040_dsp.h

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright (c) 2022 Raspberry Pi (Trading) Ltd.
+ *
```

Comment:
I suspect this copyright is wrong.

Reply:
I have copied it from the other board files, which also didn't seem to change the copyright.